### ISSUE TITLE
Cannot read property 'createCollection' of null

### ISSUE THREAD

**Issue by phantomis** [db-migrate/node-db-migrate#527]
Hi everyone

Today I was playing with this tool and running the most basic example I received this error: 

```
✘ riamaro  ~/FIF/DesCargos/cmr-cl-desc-cargos-api-mantenedor-parametros   dev ●  NODE_ENV="local" node node_modules/db-migrate/bin/db-migrate up -v
[INFO] Detected and using the projects local version of db-migrate. '/home/riamaro/FIF/DesCargos/cmr-cl-desc-cargos-api-mantenedor-parametros/node_modules/db-migrate/index.js'
[INFO] Using local settings: { driver: 'mongodb',
  database: 'reclamos-local',
  host: '172.17.0.2',
  port: '27017',
  user: 'admin',
  password: '******' }
[INFO] require: db-migrate-mongodb
[INFO] connecting
[INFO] connected
[SQL] createCollection [ 'migrations' ]
/home/riamaro/FIF/DesCargos/cmr-cl-desc-cargos-api-mantenedor-parametros/node_modules/db-migrate-mongodb/node_modules/mongodb/lib/mongo_client.js:462
            throw err
            ^

TypeError: Cannot read property 'createCollection' of null
    at /home/riamaro/FIF/DesCargos/cmr-cl-desc-cargos-api-mantenedor-parametros/node_modules/db-migrate-mongodb/index.js:349:15
    at connectCallback (/home/riamaro/FIF/DesCargos/cmr-cl-desc-cargos-api-mantenedor-parametros/node_modules/db-migrate-mongodb/node_modules/mongodb/lib/mongo_client.js:515:5)
    at /home/riamaro/FIF/DesCargos/cmr-cl-desc-cargos-api-mantenedor-parametros/node_modules/db-migrate-mongodb/node_modules/mongodb/lib/mongo_client.js:459:13
    at process._tickCallback (internal/process/next_tick.js:150:11)

```

The  example code is this: 
``` javascript
'use strict';

var dbm;
var type;
var seed;

/**
  * We receive the dbmigrate dependency from dbmigrate initially.
  * This enables us to not have to rely on NODE_PATH.
  */
exports.setup = function(options, seedLink) {
  dbm = options.dbmigrate;
  type = dbm.dataType;
  seed = seedLink;
};

exports.up = function(db) {
  return null;
};

exports.down = function(db) {
  return null;
};

/*

exports.up = db => db.createCollection('globalConfiguration')
    .then(() => {
      db.insert('globalConfiguration', { name: 'FeatureDisputeChargeMobileEnabled', value: true });
    });

exports.down = db => db.dropCollection('globalConfiguration');

*/

exports._meta = {
  "version": 1
};
```

**Comment by vjustov** [comment]
same thing here. anything i could provide to help debug the problem?

**Comment by dpkshrma** [comment]
For me it happened when mongo db connection failed

**Comment by pztrick** [comment]
I encountered this when attempting to use a MongoDB Atlas connection which uses the `mongodb+srv` prefix.

The error message is useless but means that the mongodb driver connection has failed. See: https://github.com/db-migrate/mongodb/issues/43

There is a third-party driver `db-migrate-mongodb-nesto-fork` that works with `SRV` config now. You might be able to install using `npm` but I just used the explicit Git commit. See: https://github.com/db-migrate/mongodb/issues/25#issuecomment-455915040

```
npm install --save "github:nesto-software/mongodb#17290a4"
```

```
{
    "sandbox": {
        "driver": "mongodb-nesto-fork",
        "useSrvRecord": "true",
        "host": "whatever.mongodb.net",
        "user": "root",
        "password": "pass",
        "database": "whatever-dev"
    }
}
```

Note the different `driver` and `useSrvRecord` fields.
